features/step_definitions/api-token.js

```diff
@@ -47,13 +48,15 @@ module.exports = function server() {
                 bearer: this.jwt
             }
         }).then(() => this.getJwt(this.testToken.value).then((response) => {
-            Assert.strictEqual(response.statusCode, 200);
+            Assert.oneOf(response.statusCode, [200, 401]);
```

Comment:
I think depending on which test is run, those would be the two expected response codes. A `500` would not be good/expected. Not sure if there's a better way to do this, but we also do this in the authorization test:
https://github.com/screwdriver-cd/screwdriver/blob/master/features/step_definitions/authorization.js#L58